readme.md

```diff
@@ -41,33 +26,71 @@ The documentation can be found [here](https://docs.walletbeacon.io/).
 npm i --save @airgap/beacon-sdk
 ```
 
-## Example
+## Upgrading from v1 to v2
```

Comment:
create an entry for this in the docs as well

Reply:
Done